### PR TITLE
lightning: add local backend config

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

Add `local` backend configuration.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3806
- Other reference link(s):

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -98,15 +98,28 @@ driver = "file"
...
+# region-split-size = 100_663_296
+# The number of KV pairs sent in one request in the "local" backend.
+# send-kv-pairs = 32768
+# The directory of local KV sorting in the "local" backend. If the disk
+# performance is low (such as in HDD), it is recommended to set the directory
+# on a different disk from `data-source-dir` to improve import speed.
+# sorted-kv-dir = ""
+# The concurrency that TiKV writes KV data in the "local" backend.
+# When the network transmission speed between TiDB Lightning and TiKV
+# exceeds 10 Gigabit, you can increase this value accordingly.
```
Comment (yikeke) [inline tidb-lightning/tidb-lightning-configuration.md:121]:
```suggestion
# exceeds 10 gigabytes, you can increase this value accordingly.
```
gigabit = 千兆位 gigabyte = 千兆字节 Please make sure which one is correct. @ran-huang

Comment (nullnotnil) [reply tidb-lightning/tidb-lightning-configuration.md:121]: It is correct at 10 Gigabit. Network transfer is measured in bits.

**Comment by nullnotnil** [comment]
LGTM

**Pull request merged**